### DISCUSSION TITLE
OSX 10.14.6 Mojave Compatibility

### DISCUSSION THREAD

**Discussion by FuugoClay** [keepassxreboot/keepassxc#9845] [Q&A] [answered]
It appears the last version that supported Mojave [OSX 10.14.6] is 2.7.1.

Is this due to a fundamental change (a library or something that is not available with the last "OSX" build? And, is there a way to install the latest build on Mojave?

I am not looking to start an argument about updating OS's..  I have reasons for not upgrading to later "MacOS" versions.  :-)

*I did not find any reference to this in the release notes or in the wiki or in the Discussion area.

**Comment by droidmonkey** [comment, marked as answer]
Qt stopped supporting outdated macOS. Use macports distro to gain access to custom compiled versions for your os.

**Reply by FuugoClay** [reply]
I am running 10.14.6 with xcode11.3.1 / 13.3 SDK .

Mojave is supported up through at least qt6.2.

[Qt 6.2](https://web.archive.org/web/20220122123945/https://doc.qt.io/qt-6/index.html) Qt for macOS

 The following macOS versions are supported.

Target Platform	Architecture	Build Environment
macOS 10.14, 10.15, 11	x86_64, x86_64h, and arm64	Xcode 12 (11 SDK)

![2023-09-15_12-02-35](https://github.com/keepassxreboot/keepassxc/assets/36994025/e634ea8d-343b-4c5e-b640-e8962fd45463)
![2023-09-15_12-00-27](https://github.com/keepassxreboot/keepassxc/assets/36994025/671dca00-527c-4c40-a20b-b3a1c8f6aa1e)

**Reply by droidmonkey** [reply]
First off we aren't using Qt 6, secondly it's much more complicated then just looking at a support table. You actually need to set a lower bound for compilation. We use Qt 5 delivered by homebrew which locks to supported versions of macOS.

https://doc.qt.io/qt-5/macos.html

You should just use macports like I suggested, we aren't going to do anything different to support outdated unsupported operating systems.

**Comment by jamesquilty** [comment]
@FuugoClay I'm in a similar situation to you and had the same questions. Did you make any progress on building the latest version of KeePassXC for macOS 10.14?
